python/ray/data/dataset.py

```diff
@@ -1338,7 +1339,7 @@ def to_modin(self) -> "modin.DataFrame":
 
         from modin.distributed.dataframe.pandas.partitions import (
             from_partitions)
-        pd_objs = self.to_pandas()
+        pd_objs = self.to_pandas_refs()
         return from_partitions(pd_objs, axis=0)
 
     def to_spark(self,
```

Comment:
should we apply the same limit to (to_spark)? i.e. will it incur OOM issue?

Reply:
To spark/Dask is a distributed conversion, which doesn't gather data to the driver.